w09h03/test/pgdp/filter/DGTest.java

```diff
@@ -375,9 +378,6 @@ public void test20() throws IOException {
                 Assertions.assertEquals(Color.YELLOW.getRGB(), output.getPixels().getRGB(i, j));
             }
         }
-
-        outputFrame("test20Output.png", output, "PNG");
-        outputFrame("test20Original.png", frame, "PNG");
```

Comment:
Do I understand correctly that this simply won't draw frames if tests fail?

Reply:
Frames should get saved in line 366